src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/esa/EsaIssueFinalDecisionAboutToSubmitHandlerTest.java

```diff
@@ -414,9 +409,23 @@ public void throwsExceptionIfItCannotHandleTheAppeal() {
     }
 
     @Test
-    public void givenAnIssueFinalDecisionEvent_shouldUpdateStateAndDwpStateWhenStateIsNotReadyToListOrWithFta() {
-        commonSetUp();
+    public void givenAnIssueFinalDecisionEvent_shouldUpdateIssueFinalDecisionDateToToday() {
+        callback.getCaseDetails().getCaseData().getSscsFinalDecisionCaseData().setWriteFinalDecisionPreviewDocument(documentLink);
```

Comment:
`callback.getCaseDetails().getCaseData().getSscsFinalDecisionCaseData()` that definitly could be separate var! 
and everywhere below aswell ;)

Reply:
done